ldap_jwt_auth/auth/authentication.py

```diff
@@ -52,3 +60,29 @@ def authenticate(self, user_credentials: UserCredentialsPostRequestSchema) -> No
             message = "Problem with LDAP server"
             logger.exception(message)
             raise LDAPServerError(message) from exc
+
+    def is_user_active(self, username: str) -> bool:
+        """
+        Check if the provided username is part of the active usernames.
+        :param username: The username to check.
+        :return: `True` if the user is active, `False` otherwise.
+        """
+        active_usernames = self._get_active_usernames()
+        logger.debug(len(active_usernames))
+        logger.debug(active_usernames)
+        return username in active_usernames
+
+    def _get_active_usernames(self) -> list:
+        """
+        Load the active usernames as a list from a `txt` file. It removes any leading and trailing whitespaces and does
+        not load empty lines/strings.
+        :return: The list of active usernames.
+        :raises ActiveUsernamesFileNotFoundError: If the file containing the active usernames cannot be found.
+        """
+        try:
+            with open(config.authentication.active_usernames_path, "r", encoding="utf-8") as f:
+                return [line.strip() for line in f.readlines() if line.strip()]
```

Comment:
```suggestion
            with open(config.authentication.active_usernames_path, "r", encoding="utf-8") as file:
                return [line.strip() for line in file.readlines() if line.strip()]
```
Don't mind just a habit from pylint.

Reply:
Addressed in https://github.com/ral-facilities/ldap-jwt-auth/pull/22/commits/6a98d4ced706fe884584f1da66dbb36f95b03225